docs/metadata-ui.rst

```diff
@@ -0,0 +1,190 @@
+Import metadata using the Populate Metadata script in OMERO.web
+===============================================================
+
+Description
+-----------
+
+This chapter will show how to import metadata starting from a local CSV file and ending with OMERO.tables on images or wells using the `server-side script <https://omero-guides.readthedocs.io/en/latest/scripts/docs/index.html>`__ Populate Metadata in OMERO.web. See also the workflow described in :doc:`metadata` which will give you more possibilities, such as Key-Value Pairs creation, but which is using Command Line Interface (CLI). The workflow described here is using only graphical user interface elements.
+
+This action is typically done after a successful import of images.
+
+We will show:
+
+- How to import metadata from local CSV file in a bulk manner and turn them into OMERO.tables on
+   - :ref:`Images contained in Projects/Datasets<Populatemetadatapdi>`
+   - :ref:`Wells contained in Screens/Plates<Populatemetadataspw>`
+
+- How to :ref:`create or adjust a local CSV file containing metadata<Populatemetadatacsv>` for creation of OMERO.tables containing numbers and text. 
+
+Resources
+---------
+
+-  Data: example images from
+
+   -  IDR data `idr0021-lawo-pericentriolarmaterial/experimentA <https://idr.openmicroscopy.org/webclient/?show=project-51>`_
+
+-  Annotation CSV files define the content of OMERO.tables for each image or each well.
+
+   - :download:`four-images.csv <../scripts/four-images.csv>`
+   - :download:`simple-screen.csv <../scripts/four-images.csv>`
+
+-  ``omero-metadata`` plugin (necessary for having full set of features in ``Populate Metadata`` script).
+
+   - https://pypi.org/project/omero-metadata/
+
+Setup
+-----
+
+**omero-metadata plugin installation**
+
+.. note::
+      For the best experience, the ``omero-metadata`` plugin should be installed on your OMERO.server. The ``Populate Metadata`` script tries to reuse the code of the ``omero-metadata`` plugin. If the plugin is not found on the server, the ``Populate Metadata`` script falls back on a deprecated code, with limited set of features. The ``omero-metadata`` plugin installation is typically done by the administrator of the OMERO.server.
```

Comment:
Do you reckon this should be compulsory in the OMERO.server installation instructions?

Reply:
fmpov, clear Yes. My main motivation for the Yes is the recent constant stream of external users (on the level of OMERO admins, facility managers etc.) who would like to have the plugin&script as ready as possible for their basic users to handle their metadata.

Reply:
The ansible role installs the omero-server python package
https://github.com/ome/ansible-role-omero-server/blob/5bce48f0e238ebd4b37fd1dc1f4483f3fe76acde/defaults/main.yml#L113
which has these compulsory dependencies:
https://github.com/ome/omero-cli-server/blob/e65c0d60a6c6e892fab8533eabb067c770077503/setup.py#L51-L56

You could add open a PR to add it there and see what everyone else thinks?

Reply:
@manics Opened https://github.com/ome/omero-cli-server/pull/5